src/config.ts

```diff
@@ -297,3 +298,13 @@ export const BARCODE_ALGORITHM = NonEmptyString.decode(
 
 // Application insights sampling percentage
 export const DEFAULT_APPINSIGHTS_SAMPLING_PERCENTAGE = 20;
+
+// Reviewer static login params
+export const TEST_LOGIN_FISCAL_CODE: ReadonlyArray<FiscalCode> = getRequiredENVVar(
```

Comment:
can we make these optional ?

Reply:
Optional with a default value?

Reply:
nope, without. endpoint is a no-op in case they're missing

Reply:
@gunzip how do you suggest to implement a no-op response? I've not found a suggestion on [Zalando guidelines](https://opensource.zalando.com/restful-api-guidelines/#http-status-codes-and-errors), maybe a status `304` response, but can't be applied on `POST` (following Zalando guidelines). Maybe we can return a simple Unauthorized.

Reply:
yes it's ok to return 401

Reply:
fixed in 48b864c

Reply:
Sorry, fixed in b11cff3